extension/android/executorch_android/src/main/java/org/pytorch/executorch/Module.java

```diff
@@ -52,7 +52,22 @@ public static Module load(final String modelPath, int loadMode) {
     if (!NativeLoader.isInitialized()) {
```

Comment:
Remove?

Reply:
I left the original constructor there for BC, though we could also just use a default value in the new constructor to maintain source compatibility. Are there any other ABI-style implications? What is your preference?